app/views/companies/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>
+  Information about the company with the ID of <%= @company.id %>
+</h1>
+<li><%= @company.name %></li>
+<li><%= @company.email %></li>
+<li><%= @company.active %></li>
+<%= link_to "delete", @company, method: :delete %>
```

Comment:
add o temporary javascript confirm window. For now that is good enough. But a delete with no confirmation is not good